builtin/providers/google/resource_sql_user.go

```diff
@@ -95,32 +104,41 @@ func resourceSqlUserRead(d *schema.ResourceData, meta interface{}) error {
 		return err
 	}
 
-	name := d.Get("name").(string)
-	instance := d.Get("instance").(string)
+	instanceAndName := strings.SplitN(d.Id(), ".", 2)
```

Comment:
I'm slightly nervous about using dots as a separator. Even though this set of properties is safe, I can imagine a resource that needs to be imported in a similar way but dots are acceptable characters in names for both types of resources, and then we would have an inconsistency between resources. I think a "/" would be safer, since I'd be really surprised if it ever came up as a separator (because of weirdness in self_links). What do you think?